tests/phpunit/tests/html-api/wpHtmlProcessorBreadcrumbs.php

```diff
@@ -545,4 +545,26 @@ public function test_can_seek_back_and_forth() {
 		$processor->seek( 'second' );
 		$this->assertTrue( $processor->get_attribute( 'two' ) );
 	}
+
+	/**
+	 * Ensures that breadcrumbs are properly reported after seeking backward to a location
+	 * inside an element which has been fully closed before the seek.
+	 *
+	 * @ticket {TICKET_NUMBER}
```

Comment:
Flagging the ticket to update before committing.

```suggestion
	 * @ticket {TICKET_NUMBER}
```

Reply:
Added